Example/RoboKittenTests/Templates/AutoMockable.stencil

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import SwiftyMock
+import Result
+@testable import Betterme
```

Comment:
awesome 👍 

but please remove this dependency

btw is it possible to know in stencil what module we're in? like in xcode templates?

Reply:
unfortunately I didn't found solution for that yet